x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/boost_icon.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiToken } from '@elastic/eui';
+
+import { BOOST_TYPE_TO_ICON_MAP } from './constants';
+import { BoostType } from './types';
+
+interface Props {
+  type: BoostType;
+}
+
+export const BoostIcon: React.FC<Props> = ({ type }) => {
```

Comment:
not sure if this is covered by a `mount` somewhere, but do you need a unit test for this component or nah?

Reply:
It was covered by a mount, I think, because coverage report wasn't complaining, so I was just being lazy 😅 .

Always good to have a specific test though: 8de7194265308a174610c20bb104c50465cf647a